### PR TITLE
show only image in profile activity

### LINKED ISSUE #10030: Profile Activity is laggy af
I was checking my profile activity, and it was incredibly laggy and freezing my laptop. My profile - [here](https://kodadot.xyz/ahp/u/1NFfEH3yspdEgLnhZ5QWgb7B2z5LNCAE3HUo5nDxF5MqTcj?tab=activity&buy=true&sale=false&list=false&transfer=true).

try to switch between options " buy, list, sale..."

I guess because I have more transactions there, but still, user experience is very bad here.

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA

- @prury please review
- @JustLuuuu

## Context

- [x] Ref #10030

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

### CODE REVIEW

File: components/profile/activityTab/HistoryRow.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
             class="border border-k-shade w-[3.125rem] h-[3.125rem]"
             :alt="event.Item.name"
             :src="image"
-            :animation-src="animationUrl"
+            :animation-src="!image ? animationUrl : undefined"
```
Comment (vikiival) [inline components/profile/activityTab/HistoryRow.vue:12]: isn't it better to have one computed variable (DRY) ?

**Review by vikiival — APPROVED** [review]
Otherwise LGTM

File: components/profile/activityTab/HistoryRow.vue (hunk 1 above)
Comment (daiagi) [reply components/profile/activityTab/HistoryRow.vue:12]: i don't like to have computed refs for such simple things

i feel that it creates a mental overhead of linking the ref to where it is used in the template

matter of personal preference i guess

if you disagree in this case lmk and ill change


thanks :heart:

**Review by hassnian — APPROVED** [review]

**Comment by prury** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 8.26 USD/DOT ~ 2.421 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0x2a942178bb0519b064ea80fda43ea8d88333d2d732411d6b74a4de9e664b70c8](https://polkadot.subscan.io/extrinsic/0x2a942178bb0519b064ea80fda43ea8d88333d2d732411d6b74a4de9e664b70c8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**